.github/workflows/test_install.yml

```
@@ -0,0 +1,68 @@
+name: Test Installation
+
+on:
+  workflow_dispatch:
+    inputs:
+      os:
+        type: choice
+        description: 'Operating System'
+        required: true
+        options:
+          - ubuntu-latest
+          - macos-latest
+        default: 'ubuntu-latest'
+
+jobs:
+  test-install:
+    name: test installation (${{ github.event.inputs.os }})
+    runs-on: ${{ github.event.inputs.os }}
+
+    steps:
+    - name: checkout code
+      uses: actions/checkout@v2
+
+    - name: install Spice (https://install.spiceai.org)
+      run: |
+        curl https://install.spiceai.org | /bin/bash
+
+    - name: Add Spice bin to PATH
+      run: echo "/home/runner/.spice/bin" >> $GITHUB_PATH
```

note: that path will be different on other platforms, let's use HOM env var
```suggestion
      run: echo "$HOME/.spice/bin" >> $GITHUB_PATH
```